packages/core/src/common/types.ts

```diff
@@ -102,6 +102,13 @@ export interface Threshold<T extends ThresholdValue = ThresholdValue>
   dataStreamIds?: DataStreamId[];
 }
 
+export type ThresholdStyleType = {
```

Comment:
moved the new types here, since they are referenced in both dashboard and react-components